src/DIRAC/Interfaces/API/Job.py

```diff
@@ -487,6 +486,30 @@ def setCPUTime(self, timeInSecs):
         self._addParameter(self.workflow, "CPUTime", "JDL", timeInSecs, description)
         return S_OK()
 
+    #############################################################################
+    def setPriority(self, priority: int):
+        """Helper function.
+
+        A job with a high priority has a higher probability of being matched
+        (and consequently going to "Running" state)
+
+        Example usage:
+        >>> job = Job()
+        >>> job.setPriority(2)
+
+        :param priority: priority (between 0 and 10 in default configuration)
+        """
+        kwargs = {"priority": priority}
+        if not isinstance(priority, int):
+            try:
+                priority = int(priority)
```

Comment:
What could it be if not an `int`?

Reply:
I just followed what was done in the setCPUTime method.